source/developer/developer-setup.md

```diff
@@ -33,17 +33,20 @@ Developer Machine Setup
 	4. `git clone https://github.com/<username>/platform.git`  
 	5. `cd platform`
 
-5. Install godep with `go get github.com/tools/godep`
+5. If you are running Mattermost version 3.0 or older, install godep with `go get github.com/tools/godep`
```

Comment:
No need to have this extra noise about old versions. This is a developer document. We don't support developing on old versions.